api/src/main/java/org/pmiops/workbench/db/model/UserDataUseAgreement.java

```diff
@@ -0,0 +1,129 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "user_data_use_agreement")
+public class UserDataUseAgreement {
+  private long userDataUseAgreementId;
+  private long userId;
+  private String userGivenName;
+  private String userFamilyName;
+  private String userInitials;
+  private boolean userNameOutOfDate;
+  private int dataUseAgreementSignedVersion;
+  private Timestamp completionTime;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "user_data_use_agreement_id")
+  public long getUserDataUseAgreementId() {
+    return userDataUseAgreementId;
+  }
+
+  public void setUserDataUseAgreementId(long userDataUseAgreementId) {
+    this.userDataUseAgreementId = userDataUseAgreementId;
+  }
+
+  @Column(name = "user_id")
+  public long getUserId() {
+    return userId;
+  }
+
+  public void setUserId(long userId) {
+    this.userId = userId;
+  }
+
+  @Column(name = "user_given_name")
+  public String getUserGivenName() {
+    return userGivenName;
+  }
+
+  public void setUserGivenName(String userGivenName) {
+    this.userGivenName = userGivenName;
+  }
+
+  @Column(name = "user_family_name")
+  public String getUserFamilyName() {
+    return userFamilyName;
+  }
+
+  public void setUserFamilyName(String userFamilyName) {
+    this.userFamilyName = userFamilyName;
+  }
+
+  @Column(name = "user_initials")
+  public String getUserInitials() {
+    return userInitials;
+  }
+
+  public void setUserInitials(String userInitials) {
+    this.userInitials = userInitials;
+  }
+
+  @Column(name = "user_name_out_of_date")
```

Comment:
Please put some comments about what this means. Does it cover both soft and "hard" name changes?